test/README.md

```diff
@@ -84,7 +84,7 @@ A docker tag is mandatory to avoid issues with using `latest` tag for images dep
 
 ### Adding new test images
 
-New test image paths should be placed in `./test/image_paths.txt`.
+New test images should be placed in `./test/test_images`.
```

Comment:
It's unclear to me from this how to do it, since I don't see the existing image here. Or, if I add another test, looking at this, I wouldn't know what I need to do.